concent_api/core/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db.models import Model, CharField, IntegerField, DateTimeField, BooleanField, BinaryField, ForeignKey
+
+
+class Message(Model):
+    type        = CharField(max_length=32)
```

Comment:
Przydałyby się komentarze czym jest dany model.

Reply:
You mean docstrings?

This is not a part of the public API so I don't require documentation here.

This is mostly due to time constraints and because the database structure in small Django apps is fairly obvious (it often maps directly to endpoints). We need more docs on the API level first. Only when that's done it makes sense to go deeper and start documenting the internals.

But this attitude comes largely from the fact that usually the only person that writes docs is me. Now that there are two of us, we could split this responsibility and I'm all for having more docs in the code if you can help @Jakub89 and @kbeker write them.

Reply:
I have rather meant some comments that can be useful for other developers, even something simple like 
"""This model represents message received from Golem.""" etc.
If we need we can extend it later.

Reply:
Well, I don't usually add such descriptions unless I have something substantial to write. But I'm not against them. It's fine to point it out in a review.

Personally, I prefer to put much more emphasis on having good, self-explanatory names.